.github/workflows/ci.yml

```
@@ -51,6 +51,10 @@ jobs:
           stack build --haddock --test --only-dependencies
           stack install doctest
 
+      - name: Install weeder
+        run: |
+          stack install weeder-2.4.1
```

Use one line syntax.

```suggestion
      - name: Install weeder
        run: stack install weeder-2.4.1
```